riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/RiptideAutoConfiguration.java

```diff
@@ -24,6 +24,7 @@
         "org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration",
         "org.zalando.logbook.spring.LogbookAutoConfiguration",
         "org.zalando.tracer.spring.TracerAutoConfiguration",
+        "org.zalando.tracer.spring.TracerSchedulingAutoConfiguration",
 })
 public class RiptideAutoConfiguration {
```

Comment:
To clean this up, we should remove the `AutoConfigureAfter` from the inner configuration and move the `AutoConfigureBefore` here as well.